src/main/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -534,7 +461,7 @@ public static synchronized void initializeAnyploidPLIndexToAlleleIndices(final i
         }
 
         if (ploidy <= 0) {
-            throw new IllegalStateException("The ploidy " + ploidy + " must be greater than zero");
+            throw new IllegalStateException("The ploidy must be greater than zero");
```

Comment:
I would keep the actual value in the error message.  If it's ever triggered we'll want to know.